### ISSUE TITLE
Tool fails, versions >= v0.10.0

### ISSUE THREAD

**Issue by georlav** [mfridman/tparse#63]
All versions >= v0.10.0 fail

Hello i am using your parser in some of my CIs and after your latest changes it fails with the following error. 
When i use v0.9.0 everything works.

```bash 
go test -race ./... -v -short -json --failfast | tparse -all -notests -dump

Error: ../../../go/pkg/mod/github.com/mfridman/tparse@v0.10.1/internal/app/table_failed.go:52:18: undefined: strings.Cut
Error: Process completed with exit code 2.
```

**Comment by mfridman** [comment]
Sorry about that. The short of it is _some_ flags were removed with the underlying implementation. The `-dump` flag is replace with `-follow`.

In order to make forward progress on the project I needed to refactor some bits, and `-dump` was never an ideal implementation because it was holding _all_ the JSON events in-memory.

The `-follow` is almost always what folks want. It "streams" or "follows" go test output as the tests is running without buffering that output.
